files/jsondata/L10n-CSS.json

```
@@ -84,6 +92,7 @@
     "es": "Pseudo-clases",
     "fr": "Pseudo-classes",
     "ja": "擬似クラス",
+    "ko": "가상 클래스",
```

```suggestion
    "ko": "의사 클래스",
```

Pseudo를 "의사"하고 "가상"중에 어느것으로 번역하는 것이 좋을지 고민이 됩니다. 다른 이유가 없다면 현재 MDN 한국 페이지에서 많이 사용되고 있는 의사를 사용하는것이 좋아 보입니다.

현재 MDN 한국 문서에는 의사 클래스(34 files)가 가상 클래스(10 files)보다 더 많이 사용되고 있습니다. (의사 요소(24 files) vs 가상 요소(6 files))

---

"의사"가 더 일반적으로 쓰이는것 같습니다. 👍👍